setup.py

```diff
@@ -1,58 +1,14 @@
-import os
 import pathlib
 from setuptools import setup, find_packages
-from setuptools.dist import Distribution
 import sys
 import versioneer
 
-try:
-    from wheel.bdist_wheel import bdist_wheel
-
-    HAS_WHEEL = True
-except ImportError:
-    HAS_WHEEL = False
-
-if HAS_WHEEL:
-
-    class UnidistWheel(bdist_wheel):
-        def finalize_options(self):
-            bdist_wheel.finalize_options(self)
-            self.root_is_pure = False
-
-        def get_tag(self):
-            _, _, plat = bdist_wheel.get_tag(self)
-            py = "py3"
-            abi = "none"
-            return py, abi, plat
-
-
-class UnidistDistribution(Distribution):
-    def __init__(self, *attrs):
-        Distribution.__init__(self, *attrs)
-        if HAS_WHEEL:
-            self.cmdclass["bdist_wheel"] = UnidistWheel
-
-    def is_pure(self):
-        return False
-
```

Comment:
Why is this not needed now?

Reply:
Take a look at my comment to @no-ponomarev, as well as at the issue description.

Reply:
Ok